libs/src/services/dataContracts/index.js

```diff
@@ -105,7 +105,7 @@ class AudiusContracts {
   // Special case initialization flow for UserReplicaSetManagerClient backwards compatibility
   // Until the contract is deployed and added to the data contract registry, replica set
   // operations will flow through the existing UserFactory
-  async initUserReplicaSetManagerClient () {
+  async initUserReplicaSetManagerClient (selectNewEndpointOnRetry = true) {
```

Comment:
should this be default to false?

Reply:
yeah i did this intentionally. per my comment in PR description, i wanted to do it this way for back compat so any existing libs consumers will have no functionality changes unless they explicitly set this param. wdyt?

Reply:
idt keeping back compat to a broken state (which this previously was) really makes sense, but we can change the caller to be 'false' instead if that's preferable?

Reply:
am testing this rq btw

Reply:
don't mind, just wanted to keep impact of this PR minimal as the immediate requirement is for chain of trust. am open to changing caller to pass false but would like to do that separately if cool with you

Reply:
eh i can do that in this PR as well since we know how simple the code change is

Reply:
yeah i think it is unrelated to chain of trust entirely

Reply:
flipped default to false, agree with you